app/services/katello/pulp3/migration_switchover.rb

```diff
@@ -0,0 +1,73 @@
+require 'pulp_2to3_migration_client'
+
+module Katello
+  module Pulp3
+    class SwitchoverError < StandardError; end
+
+    class MigrationSwitchover
+      def initialize(*argv)
+        @migration = Katello::Pulp3::Migration.new(*argv)
+      end
+
+      def content_types
+        @migration.content_types_for_migration
+      end
+
+      def run
+        check_already_migrated_content
+        cleanup_v1_docker_tags
+        migrated_content_type_check
+        combine_duplicate_docker_tags
```

Comment:
I've tested this out on a machine with hundreds of repos and content views.  `combine_duplicate_docker_tags` takes about 3 hours to run while the other tasks take under a minute each.